browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -72,7 +72,7 @@ const char token_list_json[] = R"(
    }
   })";
 
-const char ropsten_list_json[] = R"(
+const char goerli_list_json[] = R"(
```

Comment:
For context, this was previously named incorrectly  - the chain ID 0x2a refers to the Kovan network.

Reply:
182570408a1f25ab2731ef5f283b918df9b9f956-refs/branch-heads/5249_91@{#6

Reply:
What is that